yaml/array-exporter/edge/hpe-array-exporter-secret.yaml

```diff
@@ -15,9 +15,9 @@
 kind: Secret
 apiVersion: v1
 metadata:
-  name: my-array-secret
+  name: hpe-backend
   namespace: hpe-storage
 stringData:
   address: 192.168.1.1
-  username: myuser
-  password: mypassword
+  username: admin
```

Comment:
Do we want to put credentials here?  Seems like they'll need to be edited regardless.

Reply:
This is for consistency of the previous examples.